checkQC/app.py

```diff
@@ -57,29 +58,30 @@ def configure_and_run(self):
 
         :returns: The reports of the application as a dict
         """
-        try:
-            config = ConfigFactory.from_config_path(self._config_file)
-            parser_configurations = config.get("parser_configurations", None)
-            run_type_recognizer = RunTypeRecognizer(runfolder=self._runfolder)
-            instrument_and_reagent_version = run_type_recognizer.instrument_and_reagent_version()
-
-            # TODO For now assume symmetric read lengths
-            both_read_lengths = run_type_recognizer.read_length()
-            read_length = int(both_read_lengths.split("-")[0])
-            handler_config = config.get_handler_configs(instrument_and_reagent_version, read_length)
-
-            run_type_summary = RunTypeSummarizer.summarize(instrument_and_reagent_version, both_read_lengths, handler_config)
-
-            qc_engine = QCEngine(runfolder=self._runfolder,
-                                 parser_configurations=parser_configurations,
-                                 handler_config=handler_config)
-            reports = qc_engine.run()
-            reports["run_summary"] = run_type_summary
-            self.exit_status = qc_engine.exit_status
-            return reports
-        except CheckQCException as e:
-            log.error(e)
-            self.exit_status = 1
+        config = ConfigFactory.from_config_path(self._config_file)
+        parser_configurations = config.get("parser_configurations", None)
+
+        if not Path(self._runfolder).is_dir():
+            raise RunfolderNotFoundError("Could not find runfolder: {} are you "
+                                         "sure the path is correct".format(self._runfolder))
```

Comment:
Not sure how nitpicky we need to get about grammar in error messages but if you're in a nitpicky mood... capitalize "are" and add a question mark.

Reply:
:+1: